Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -294,7 +340,18 @@ open class ComposerVC: _ViewController,
             for: .touchUpInside
         )
         
+        channelController?.delegate = self
+        
         setupAttachmentsView()
+        
+        cooldownTracker.onChange = { [weak self] currentTime in
+            guard currentTime != 0 && self?.content.state != .edit else {
+                self?.content.resetSlowMode()
+                return
+            }
+            
+            self?.content.slowdownMode(cooldown: currentTime)
```

Comment:
Do we even need `resetSlowMode`?
```suggestion
            let cooldown = // check `.edit` mode or take current time
            
            self?.content.slowdownMode(cooldown: cooldown)
```